moto/ec2/responses/vpcs.py

```diff
@@ -38,6 +38,50 @@ def describe_vpc_attribute(self):
         template = self.response_template(DESCRIBE_VPC_ATTRIBUTE_RESPONSE)
         return template.render(vpc_id=vpc_id, attribute=attribute, value=value)
 
+    def describe_vpc_classic_link_dns_support(self):
+        vpc_ids = self._get_multi_param('VpcIds')
+        filters = filters_from_querystring(self.querystring)
+        vpcs = self.ec2_backend.get_all_vpcs(vpc_ids=vpc_ids, filters=filters)
+        doc_date = '2013-10-15' if 'Boto/' in self.headers.get('user-agent', '') else '2016-11-15'
```

Comment:
We have six matches of this line in this file - could we create a small util function to avoid code duplication?

Reply:
Yes, I noticed that as well, I’ll take a look. Maybe I can move the ‘doc_date’ to the ‘__init__’, but for the others I’d say filters could be a nice candidate. I haven’t taken into account any specific filters for the routes added. If there are, then for future sake we might need to stick with this. Or the future refactoring might become a bitch if we consolidate it.

Reply:
Not sure I made myself clear. I was thinking of something very simple, like:
```
    def _get_doc_date(self):
        return '2013-10-15' if 'Boto/' in self.headers.get('user-agent', '') else '2016-11-15'

    def describe_vpc_classic_link_dns_support(self):
        vpc_ids = self._get_multi_param('VpcIds')
        filters = filters_from_querystring(self.querystring)
        vpcs = self.ec2_backend.get_all_vpcs(vpc_ids=vpc_ids, filters=filters)
        doc_date = self._get_doc_date()
        ...
```

Reply:
Aah, alright. Yes definitely can do. Will create a commit asap.